packages/cli/src/workflow-execute-additional-data.ts

```diff
@@ -880,7 +880,7 @@ async function executeWorkflow(
 		}
 
 		// remove execution from active executions
-		activeExecutions.remove(executionId, fullRunData);
+		activeExecutions.finishExecution(executionId, fullRunData);
```

Comment:
The comment above is incomplete? Not sure this adds much - I'd rather the reader find the `finally` clause.